packages/ember-metal/tests/libraries_test.js

```diff
@@ -33,6 +34,8 @@ moduleFor('Libraries registry', class extends AbstractTestCase {
   ['@test only the first registration of a library is stored'](assert) {
     assert.expect(3);
 
+    // overwrite warn to supress the double registration warning (see https://github.com/emberjs/ember.js/issues/16391)
+    setDebugFunction('warn', noop);
```

Comment:
@rwjblue do we need to set it back after the test, or is that automatically handled by the test harness?

Reply:
It’s already resetting in the afterEach of this module

Reply:
Yup, that's why I didn't reset it afterwards. :)